src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER = 5;
+    private static final double DEFAULT_RADIUS = 10.0;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
```

Comment:
Random and ColorSupplier should be class-level variable

Reply:
fixed